projects/packages/sync/src/modules/class-users.php

```diff
@@ -501,17 +501,20 @@ public function save_user_handler( $user_id, $old_user_data = null ) {
 			}
 		}
 
-		/**
-		 * Fires when the client needs to sync an updated user.
-		 *
-		 * @since 1.6.3
-		 * @since-jetpack 4.2.0
-		 *
-		 * @param \WP_User The WP_User object
-		 * @param array    State - New since 5.8.0
-		 */
-		do_action( 'jetpack_sync_save_user', $user_id, $this->get_flags( $user_id ) );
-		$this->clear_flags( $user_id );
+		if ( isset( $this->flags[ $user_id ] ) ) {
```

Comment:
I realized that when the user is updated, eg the user URL, no user flags are set, therefore this needs another approach.

Reply:
Fixed in [341cb12](https://github.com/Automattic/jetpack/pull/28385/commits/341cb12e20fbf7f394be1ff1f6a4002e0cc865d2)